subprojects/core/src/test/groovy/org/gradle/api/internal/tasks/execution/ExecuteActionsTaskExecuterTest.groovy

```diff
@@ -208,7 +208,6 @@ class ExecuteActionsTaskExecuterTest extends Specification {
         interaction {
             0 * action1._
             0 * action2._
-            0 * executionContext._
```

Comment:
❓ Is it safe to remove this?

Reply:
I updated this in https://github.com/gradle/gradle/pull/15582 to use a Mock for executionContext instead and left this verification as-is.